pages/nettest.js

```diff
@@ -151,18 +151,10 @@ export default class extends React.Component {
         </Head>
         <OONIRunHero href={'https://run.ooni.io'} />
         <Container p={4}>
-          <Text pt={2}>Measure internet censorship by running OONI Probe.</Text>
 
-          <Heading pt={2} h={2}>You have the app</Heading>
-          <Text pt={2} pb={3}>If you have the OONI Probe app installed, click Run to start the test.</Text>
```

Comment:
Again I that all this text should be left because of what I mentioned below.

Reply:
OK, thanks.